great_expectations/core/suite_parameters.py

```diff
@@ -135,11 +133,8 @@ def get_parser(self):
             # or use provided pyparsing_common.number, but convert back to str:
             # fnumber = ppc.number().addParseAction(lambda t: str(t[0]))
             fnumber = Regex(r"[+-]?(?:\d+|\.\d+)(?:\.\d+)?(?:[eE][+-]?\d+)?")
-            ge_urn = Combine(
-                Literal("urn:great_expectations:") + Word(alphas, f"{alphanums}_$:?=%.&")
-            )
             variable = Word(alphas, f"{alphanums}_$")
-            ident = ge_urn | variable
+            ident = variable
```

Comment:
The naming here feels a little difficult to reason about - is there any way we can improve the readability? I realize this isn't a result of your PR so please feel free to ignore

Reply:
I think `ident` was supposed to be short for `identifier`. But no need to even keep it around. Just updated to use `variable` directly below. Thanks!